player.py

```diff
@@ -0,0 +1,16 @@
+# Parent class for different player types, each with different moves
+# No parameters
+class Player:
```

Comment:
Some of the changes I suggested earlier would require expanding this class to have more functions. That is, a separation of `getGuess` and `callsBluff`.